include/finufft/test_defs.h

```diff
@@ -16,7 +16,6 @@
 
 // convenient private finufft internals
 #include <finufft/finufft_core.h>
-#include <finufft/utils.h>
```

Comment:
In this `test_defs.h`, on windows it has dependency on `rand_r` defined in `finufft_utils.hpp`. For linux/Mac, `rand_r` is included in `cstdlib` in this header.